content/en/docs/01/resources/install-config.yaml

```
@@ -1,43 +1,30 @@
 apiVersion: v1
 baseDomain: openshift.ch
 compute:
-- architecture: amd64
-  hyperthreading: Enabled
-  name: worker
+- name: worker
   platform:
     aws:
       type: m5.2xlarge
       zones:
-      - eu-north-1a
+      - us-east-1a
   replicas: 3
 controlPlane:
-  architecture: amd64
-  hyperthreading: Enabled
   name: master
   platform:
     aws:
-      type: m5.xlarge
+      type: m6i.xlarge
   replicas: 3
 metadata:
-  creationTimestamp: null
-  name: <username>-ops-training
+  name: <username>-ops-training.openshift.ch
```

```suggestion
  name: <username>-ops-training
```
The full cluster name is `<metadata.name>.<baseDomain>`.

---

Thanks!